tools/install_bazel.sh

```diff
@@ -6,9 +6,18 @@ echo "Checking if Bazel ${BAZEL_VERSION} needs to be installed..."
 if [[ $(command -v bazel) && "$(bazel version | grep 'label' | cut -d ' ' -f 3)" =~ ${BAZEL_VERSION} ]]; then
   echo "Bazel ${BAZEL_VERSION} already installed, skipping fetch."
 else
-  wget -nv https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
-  chmod +x bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
-  sudo -E ./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
-  sudo -E mv /usr/local/bin/bazel /usr/bin
-  rm bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+  if [ "$(uname -m)" = "aarch64" ]; then
+     echo "Install Bazel on arm64: ......"
+     apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y curl unzip zip build-essential openjdk-11-jdk
+     echo "deb https://download.opensuse.org/repositories/home:/mrostecki:/bazel/xUbuntu_20.04/ /" > /etc/apt/sources.list.d/bazel.list
+     curl -L https://download.opensuse.org/repositories/home:/mrostecki:/bazel/xUbuntu_20.04/Release.key | apt-key add -
+     apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y bazel
```

Comment:
In the `else` section below, `sudo` is used and this script can be run by normal users. I think we should add `sudo` here as well.

```suggestion
     sudo apt-get update && sudo DEBIAN_FRONTEND=noninteractive apt-get install -y curl unzip zip build-essential openjdk-11-jdk
     sudo sh -c 'echo "deb https://download.opensuse.org/repositories/home:/mrostecki:/bazel/xUbuntu_20.04/ /" > /etc/apt/sources.list.d/bazel.list'
     curl -L https://download.opensuse.org/repositories/home:/mrostecki:/bazel/xUbuntu_20.04/Release.key | sudo apt-key add -
     sudo apt-get update && sudo DEBIAN_FRONTEND=noninteractive apt-get install -y bazel
```

Reply:
Does `sudo DEBIAN_FRONTEND=... apt-get ...` work as intended? I tend to do something more like `DEBIAN_FRONTEND=... sudo -E ...` to pass the environment variable correctly.

Reply:
Yes, you are right. I will fix my comment.

Reply:
Updated to your comments, thanks a lot.